common/browser_context.go

```diff
@@ -71,24 +71,25 @@ func NewBrowserContext(
 func (b *BrowserContext) AddCookies(cookies goja.Value) {
 	b.logger.Debugf("BrowserContext:AddCookies", "bctxid:%v", b.id)
 
-	var cookieParams []network.CookieParam
+	cookieParams := []network.CookieParam{}
```

Comment:
Can we catch the errors earlier so that the happy path is left aligned as much as possible. What i mean by that is doing something like:

```
if cookies == nil || goja.IsUndefined(cookies) || goja.IsNull(cookies) {
   // k6ext.Panic(
}

// happy path
```

Reply:
There is also `gojaValueExists` helper :)

Reply:
Btw, @zucchinho, This `var cookieParams []network.CookieParam` was OK; no need to change that.